src/CoreShop/Component/Order/Transformer/AbstractCartToSaleTransformer.php

```diff
@@ -158,7 +149,7 @@ public function transformSale(ProposalInterface $cart, ProposalInterface $sale,
         $sale->setPublished(true);
         $sale->setParent($orderFolder);
         $sale->setCustomer($cart->getCustomer());
-        $sale->setSaleLanguage($this->localeContext->getLocaleCode());
+        $sale->setSaleLanguage($cart->getLocaleCode());
```

Comment:
existing carts never gets the locale applied, so this could lead to some troubles here. we should add the locale to existing carts in the migration process.

Reply:
migration added https://github.com/coreshop/CoreShop/pull/399/files#diff-cfdd1e300775b2ffa2224f3bc9b0294dR23